test/unit/calculators/energy_grants_calculator_test.rb

```diff
@@ -19,8 +19,10 @@ class EnergyGrantsCalculatorTest < ActiveSupport::TestCase
     end
 
     context '#age_variant' do
-      should 'return :winter_fuel_payment if date of birth before 05-07-1951' do
-        @calculator.date_of_birth = Date.new(1951, 7, 5) - 1
+      should 'return is eligible for a :winter_fuel_payment if the date of birth is before the winter_fuel_payment_threshold date' do
+        @calculator.date_of_birth = Date.yesterday
+        @calculator.stubs(:winter_fuel_payment_threshold).returns(Date.today)
+
```

Comment:
Did you mean to add this test in this commit? I think I would've expected it to be in a3b759144e17a8fe7b80c4eaa4f84d97d3e510f9.